src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -106,6 +111,9 @@ String getHost() {
         return this.host;
     }
 
+    @Nullable
+    Map<String, ?> getOptions() { return this.options; }
```

Comment:
Why `Map<String, ?>` and not `Map<String, String>`? Allowing objects as values introduce ambiguity over value accepted types. One could configure `options.put("statement_timeout", Duration.ofMinutes(1));` and expect this to work.

Reply:
That's true. The intention was to allow values of Integer, Long etc. but you're totally right. I'll change it to <String, String>